modules/checkout/config/schema/commerce_checkout.schema.yml

```diff
@@ -70,6 +70,9 @@ commerce_checkout.commerce_checkout_pane.login:
 commerce_checkout.commerce_checkout_pane.order_summary:
   type: commerce_checkout_pane_configuration
   mapping:
+    use_views:
```

Comment:
Can't we just infer use_views from an empty 'view' key? We usually do it that way.